app/models/user.rb

```diff
@@ -0,0 +1,10 @@
+class User < ApplicationRecord
+  has_many :comments
+  has_many :posts
+  has_many :likes
+  validates :name, presence: true
```

Comment:
- [ ] Please make sure to set the foreign key in the has-many part of the one-to-many relationship which will look something like `, foreign_key: 'author_id'`